stdlib/influxdata/influxdb/from_test.go

```diff
@@ -1,714 +0,0 @@
-package influxdb_test
```

Comment:
It seems like there should be at least some testing in here, even if `from` doesn't match to an actual source in vanilla Flux, and therefore cannot be actually executed.  How about a marshaling test, similar to the other operations?

Reply:
I removed marshaling intentionally, because I thought that this kind of from shouldn't ever come in a spec from the API. Should I put in place again?

Reply:
I mean, this from doesn't do anything 😅 

I can add a test for `NewQuery`, even if not even `Bucket` is required :)

Reply:
Yeah, I think just having a super-simple `NewQuery` test would be good, to show the Flux text produces a Spec.